src/azure-cli/azure/cli/command_modules/eventgrid/_params.py

```diff
@@ -328,18 +363,28 @@ def load_arguments(self, _):    # pylint: disable=too-many-statements
         c.argument('azure_active_directory_tenant_id', help="The Azure Active Directory Tenant Id to get the access token that will be included as the bearer token in delivery requests. Applicable only for webhook as a destination")
         c.argument('azure_active_directory_application_id_or_uri', help="The Azure Active Directory Application Id or Uri to get the access token that will be included as the bearer token in delivery requests. Applicable only for webhook as a destination")
         c.argument('resource_group_name', arg_type=resource_group_name_type)
+        c.argument('enable_advanced_filtering_on_arrays', is_preview=True, arg_type=get_three_state_flag(),
+                   options_list=['--enable-advanced-filtering-on-arrays', '--enable-af-arr'], arg_group="Filtering",
+                   help="Allows advanced filters to be evaluated against an array of values instead of expecting a singular value.")
```

Comment:
This parameter has already be defined in `self.argument_context('eventgrid')` so you don't need to redefine in any sub command of `eventgrid`~

Reply:
> Can you add tests for new added parameters?